libr/util/zip.c

```diff
@@ -23,7 +23,16 @@ static const char *gzerr(int n) {
 	return errors[n];
 }
 
-R_API ut8 *r_inflate(const ut8 *src, int srcLen, int *srcConsumed, int *dstLen) {
+/**
+ * \brief inflate zlib compressed or gzipped.
+ * \param src source compressed bytes
+ * \param srcLen source bytes length
+ * \param srcConsumed comsumed source bytes length
+ * \param dstLen uncompressed uncompressed bytes length
+ * \param controls the size of the history buffer (or “window size”), and what header and trailer format is expected.
+ * \return ptr to uncompressed
+*/
+static ut8 *r_inflatew(const ut8 *src, int srcLen, int *srcConsumed, int *dstLen, int wbits) {
```

Comment:
Some things that can be improved here:

* we dont use doxygen comments, so feel free to remove them
* Dont use camelcase
* buffers length should use size_t. int have sign and size problems

Reply:
I think I took care of this. Unless I missed a buffer length somewhere.